.settings/tasks.json

```diff
@@ -0,0 +1,23 @@
+// Available variables which can be used inside of strings.
+// ${workspaceRoot}: the root folder of the team
+// ${file}: the current opened file
+// ${fileBasename}: the current opened file's basename
+// ${fileDirname}: the current opened file's dirname
+// ${fileExtname}: the current opened file's extension
+// ${cwd}: the current working directory of the spawned process

```

Comment:
Is it only VSCode that uses this `tasks.json`? I know that VSCode permits comments in json, but does anything else consume this?